src/main/webapp/trip/tripDetails.html

```diff
@@ -2,46 +2,126 @@
+                      </div>
+                    </div>
+        
+                    <div class="selected" onclick="select()">
+                      Select a Participant
+                    </div>
+                  </div> -->
+
+                <h3 class="trip-item-or">or</h3>
+                <form action="/NewPeople" method="POST">
+                    <div class="inputBox">
+                        <input name="text-input-participants" type="text" placeholder="e.g. Patrick">
+                        <input type="submit" value="add">
+                        
+                    </div>
+                </form>
+
+                <!-- Point of interest -->
+                <h3 class="trip-item">💚 Add points of interest</h3>
+                <div class="select-box">
+                    <div class="options-container">
+                      <div class="option">
+                        <input type="radio" class="radio" id="bc" name="visit"/>
+                        <label for="bc">Yoho Natioanl Park</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="alberta" name="visit" />
+                        <label for="alberta">Banff National Park</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="qc" name="visit" />
+                        <label for="qc">Quebec City</label>
+                      </div>
+                    </div>
                     
+                    <div class="selected" onclick="select()">
+                      Select a place to visit
+                    </div>
                 </div>
-            </form>
-            <h3 class="home-subtitle">Add points of interest</h3>
-            <form action="/NewEvent" method="POST">
-                <div class="inputBox">
-                    <input name="text-input-points" type="text" placeholder="e.g. Canada place" required="required">
-                    <input type="submit" value="add">
-                </div>
-            </form>
-            <h3 class="home-subtitle">Add transaction</h3>
-            <form action="/NewSpend" method="POST">
-                <div class="inputBox">
-                    <input name="text-input-points" type="text" placeholder="" required="required">
-                    <input type="submit" value="add">
-                </div>
-            </form>
+
+                <h3 class="trip-item-or">or</h3>
+                <form action="/NewEvent" method="POST">
+                    <div class="inputBox">
+                        <input name="text-input-points" type="text" placeholder="e.g. Canada place" required="required">
+                        <input type="submit" value="add">
+                    </div>
+                </form>
+
+                <!-- Transactions -->
+                <h3 class="trip-item">💚 Add transaction</h3>
```

Comment:
I think this could be moved to the trip page since users likely won't have transactions when planning and they'll need this section on the trip page since they can't return to the create trip page.

Reply:
Create trip page only has trip name, dates and total budget. I dont think it's a good place to have transaction details there. Or we can move the transaction to the dashboard page

Reply:
Oh i thought this was the create trip page my bad.